### PR TITLE
DeltaMax's Mania Key Smasher Wiki Page

### PR DESCRIPTION
**Pull request by fajar13k**
Start DMX:MKS wiki page.

---

### CODE REVIEW

File: wiki/Tournaments/DMX/3/en.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+| Finals | 2018-12-15/2018-08-16 |
+| Grand Finals | 2018-12-22 |
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for DeltaMax's Mania Key Smasher winners. It is possible that these items change with every installment of the DeltaMax's Tournaments.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | unique profile badge |
```
Comment (TPGPL) [inline wiki/Tournaments/DMX/3/en.md]: can be `Unique profile badge`

**Comment by fajar13k** [comment]
Ready

File: wiki/Tournaments/DMX/3/en.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+### Saturday, 24 November 2018
+
+| Match ID | Player A |  |  | Player B | Match Time |
+| :-: | --: | :-: | :-: | :-- | :-: |
+| F5 | Shiyui- | ![][flag_PH] | ![][flag_CL] | WalterToro | **03:00 UTC** |
+| H2 | SamsonNote7 | ![][flag_HK] | ![][flag_MY] | \[MY\]xRay | **09:00 UTC** |
+| E2 | ZayyKen | ![][flag_FR] | ![][flag_AU] | Spartan- | **10:00 UTC** |
+| E3 | Spartan- | ![][flag_AU] | ![][flag_FR] | DemonWaves | **11:00 UTC** |
+| F2 | Shiyui- | ![][flag_PH] | ![][flag_PL] | SitekX | **11:00 UTC** |
+| G3 | cheewe10 | ![][flag_MY] | ![][flag_TH] | HowToPlayLN | **11:00 UTC** |
```
Comment (WalterToro) [inline wiki/Tournaments/DMX/3/en.md]: cheewe10 > cheewee10 (repeat on other occurrences)

File: wiki/Tournaments/DMX/3/en.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+| H6 | Edvo | ![][flag_VE] | ![][flag_MY] | \[MY\]xRay | **12:00 UTC** |
+| H4 | Edvo | ![][flag_VE] | ![][flag_HK] | SamsonNote7 | **13:00 UTC** |
+| D4 | Hlimak | ![][flag_PL] | ![][flag_DE] | Orange127 | **14:00 UTC** |
+| E4 | 2fast4you98 | ![][flag_NL] | ![][flag_FR] | ZayyKen | **15:00 UTC** |
+| F6 | Arkitev | ![][flag_PL] | ![][flag_PL] | SitekX | **15:00 UTC** |
+| G4 | CribobFangirl | ![][flag_IT] | ![][flag_IT] | Kiraz | **15:00 UTC** |
+
+-----------------------
+
+## Participants
```
Comment (WalterToro) [inline wiki/Tournaments/DMX/3/en.md]: Participants and groups over schedule

**Pull request merged**